templates/cases/case_read.haml

```diff
@@ -169,7 +169,10 @@
                 .contact-field-label.col-sm-6
                   [[ urn.scheme ]]
                 .contact-field-value.col-sm-6
-                  [[ ]]
+                  - if site_redact_urns
+                    [[ "\u2022".repeat(urn.path.length) ]]
```

Comment:
this should be fixed length of 8 same as rapidpro - some URNs are really long and we don't want to leak even the length of the URN